src/Components/Books.js

```diff
@@ -1,12 +1,32 @@
 import React from 'react';
 import Navbar from './Navbar';
+import AddBooks from './AddBook';
 
 function Books() {
   return (
     <div>
       <Navbar />
       <div className="container">
-        <h2>List of availabe books!</h2>
+        <div className="bookList">
+          <br />
+          <div className="card bg-light text-dark">
+            <div className="card-body">
+              <h6> Action</h6>
+              <h4>Book Name</h4>
+              <h6> Author Name</h6>
+              <button type="button" className="btn btn-light">Comment</button>
+              {' '}
+              |
+              <button type="button" className="btn btn-light">Remove</button>
+              {' '}
+              |
+              <button type="button" className="btn btn-light">Edit</button>
+            </div>
+          </div>
+        </div>
```

Comment:
- [ ] As the previous reviewer mentioned, you need a `Book` component it will be used to render a list of books, each time you create a book component and give it data as props, kindly implement this as it is required. 😁